docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\">┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n",
+       "┃<span style=\"font-weight: bold\"> Greater(TRANSACTION_AMT, 0) </span>┃<span style=\"font-weight: bold\"> Greater(TRANSACTION_AMT, 0)_count </span>┃\n",
+       "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩\n",
+       "│ <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">boolean</span>                     │ <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">int64</span>                             │\n",
+       "├─────────────────────────────┼───────────────────────────────────┤\n",
+       "│ True                        │                          <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">20669810</span> │\n",
+       "│ False                       │                           <span style=\"color: #008080; text-decoration-color: #008080; font-weight: bold\">1018183</span> │\n",
+       "└─────────────────────────────┴───────────────────────────────────┘\n",
+       "</pre>\n"
+      ],
+      "text/plain": [
+       "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n",
+       "┃\u001b[1m \u001b[0m\u001b[1mGreater(TRANSACTION_AMT, 0)\u001b[0m\u001b[1m \u001b[0m┃\u001b[1m \u001b[0m\u001b[1mGreater(TRANSACTION_AMT, 0)_count\u001b[0m\u001b[1m \u001b[0m┃\n",
+       "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩\n",
+       "│ \u001b[2mboolean\u001b[0m                     │ \u001b[2mint64\u001b[0m                             │\n",
+       "├─────────────────────────────┼───────────────────────────────────┤\n",
+       "│ True                        │                          \u001b[1;36m20669810\u001b[0m │\n",
+       "│ False                       │                           \u001b[1;36m1018183\u001b[0m │\n",
+       "└─────────────────────────────┴───────────────────────────────────┘"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "above_zero = cleaned.TRANSACTION_AMT > 0\n",
+    "cleaned = cleaned[above_zero]\n",
+    "above_zero.value_counts()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding Features\n",
+    "\n",
+    "Now that the data is cleaned up to a usable format, let's add some features.\n",
+    "\n",
+    "First, drop the raw columns that we already cleaned up and we don't need anymore.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, it's useful to categorize donations by size, placing them into buckets."
```

Comment:
I would remove this 'First' since you said first just above this line too. Maybe saying 'It's usefull to ...' instead :)

Reply:
lol, this was me adding/reordering notebook cells willy nilly so all the joiner/ordering words now make no sense. Good catch!